tasty-plutus/src/Test/Tasty/Plutus/Context.hs

```diff
@@ -87,7 +87,7 @@ import Plutus.V1.Ledger.Value (Value)
 import PlutusTx.Builtins (BuiltinData)
 import PlutusTx.IsData.Class (ToData (toBuiltinData))
 import Test.Tasty.Plutus.Internal.Context (
-  ContextBuilder (ContextBuilder),
+  ContextBuilder (..),
```

Comment:
Wildcard imports are against the style guide.

Reply:
So _exports_ are allowed but not imports?